website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-generating-documentation.md

```diff
@@ -6,7 +6,12 @@ description: "Automatically generate project documentation as you run jobs."
 
 ## Enabling documentation
 
-Documentation can be enabled for a job in dbt Cloud by checking the "Generate docs?" checkbox on the Job Settings page, as shown below. Once this setting is enabled, subsequent runs of the job will automatically include a step to generate documentation.
+You can enable Documentation for a job in dbt Cloud when you edit your job settings or create a new job. Once you enable this setting, subsequent runs of the job will automatically include a step to generate documentation.
```

Comment:
Are we enabling documentation for a project? Is only one job designated to generate docs?